docker/Dockerfile.arm64

```diff
@@ -1,5 +1,5 @@
-FROM alpine:3.15 AS builder
-RUN apk add --no-cache 'crystal=1.2.2-r0' shards sqlite-static yaml-static yaml-dev libxml2-dev zlib-static openssl-libs-static openssl-dev musl-dev
+FROM alpine:edge AS builder
```

Comment:
could we not jump to edge, to avoid accidentally breaking images if Crystal 1.5 releases and invidious doesn't work with it?

Reply:
You can't have crystal 1.4.0 without using edge.

Reply:
And if crystal 1.5 comes out, the docker image will just fail to build. That's the point of specifying a version in `apk add`.

Reply:
We can *technically* just install Crystal from a tarball, or just COPY from an existing image.

Otherwise, why not just use: https://hub.docker.com/r/crystallang/crystal and use `1.4.0-alpine`

Reply:
> We can _technically_ just install Crystal from a tarball, or just COPY from an existing image.
> 
> Otherwise, why not just use: [hub.docker.com/r/crystallang/crystal](https://hub.docker.com/r/crystallang/crystal) and use `1.4.0-alpine`

This is the ARM dockerfile, not the intel x86.

Reply:
>This is the ARM dockerfile, not the intel x86 one.

Yeah I just noticed, might bad. We should ask them to make an arm image.